### ISSUE TITLE
Need option to use old databases

### ISSUE THREAD

**Issue by Adamtaranto** [kdmurray91/lpipy#5]
Might be easier to use old gi2taxid map and taxonomy dump files that match the blast database version that was used to create results, rather than figuring out changes that have occurred between versions.

**Comment by kdm9** [comment]
Yeah, that's a fair point. I'll add a URL parameter, which would allow for the use of pre-downloaded files. Versioning them internally sounds too much a pain in the arse.
